api/src/database/system-data/fields/users.yaml

```diff
@@ -150,3 +150,17 @@ fields:
     display: datetime
     display_options:
       relative: true
+
+  - field: provider
```

Comment:
We should make these readonly by default at all times. The local driver is the default, and the other providers will be set during signup through the external provider